community-roadmap/backend/README.md

```diff
@@ -0,0 +1,60 @@
+# Community Roadmap for Zendesk Help Center
+
+This project contains a service that maps Zendesk community posts to product roadmap items. The service is built as a Cloudflare Worker with a Cloudflare KV data store.
+
+## Development
+
+The service can be run locally using Cloudflare's `wrangler` tool.
+
+Installation:
+```
+brew install wrangler
+```
+
+Run:
+```
+wrangler dev --env dev
+```
+or just:
+```
+npm start
+```
+
+## Test
+
+Tests are created using Postman (and the CLI runner `newman`).
+
+Installation:
+```
+brew install newman
+```
+
+The tests makes use of both pre-request scripts and test scripts in the Postman collection.
```

Comment:
This is the first mention of the Postman collection, but the README sort of implies I should be aware of it. Does this collection already exist? Do I need to fork it to my own Postman account?

```suggestion
The tests use pre-request scripts and test scripts in the Postman collection.
```